src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -1281,49 +1344,59 @@ namespace Google.Example.Library.V1.Samples
-//   title: test maps in response handling
-//   description: When there is a very long description for a sample file we can split
-//     the description into multiple lines and the samples will be just fine
-//   usage: dotnet run
+//   title: GetBigBook: 'War and Peace'
+//   description: Testing resource name overlap
+//   usage: dotnet run [--shelf Novel] [--big_book_name "War and Peace"]
 
-// [START sample]
+using CommandLine;
+
+// [START hopper]
 namespace Google.Example.Library.V1.Samples
 {
     using Google.Example.Library.V1;
     using System;
-    using System.Threading.Tasks;
 
-    public class GetBookAsyncRequestAsyncTestOnSuccessMap
+    public class GetBigBookAsyncLongRunningFlattenedAsyncPollUntilCompleteWap2
     {
         /// <summary>
-        /// When there is a very long description for a sample file we can split
-        /// the description into multiple lines and the samples will be just fine
+        /// Testing resource name overlap
         /// </summary>
-        public static async Task SampleGetBookAsync()
+        /// <param name="shelf">Test word wrapping for long lines. This is a long comment. The name of the
+        /// shelf to retrieve the big book from.</param>
+        /// <param name="bigBookName">The name of the book.</param>
+        public static async Task SampleGetBigBookAsync(string shelf, string bigBookName)
         {
             LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
-            GetBookRequest request = new GetBookRequest
-            {
-                BookName = new BookName("[SHELF_ID]", "[BOOK_ID]"),
-            };
-            Book response = await libraryServiceClient.GetBookAsync(request);
-            string intKeyVal = response.MapStringValue[123];
-            string booleanKeyVal = response.MapBoolKey[true];
-            int mapValueField = response.MapMessageValue["key"].Field;
-            Console.WriteLine($"Test printing map fields: {response.MapListValueValue["quoted_key"]}");
-            Console.WriteLine($"Test printing enum fields of a map value: {response.MapMessageValue["key"].Alignment}");
+            // string shelf = "Novel"
+            // string bigBookName = "War and Peace"
+            BookName name = new BookName(shelf, bigBookName);
+            // Poll until the returned long-running operation is complete
+            Operation<Book, GetBigBookMetadata> operation = await libraryServiceClient.GetBigBookAsync(name);
+            Book response = (await operation.PollUntilCompletedAsync()).Result;
+            Console.WriteLine(response);
         }
-        // [END sample]
+        // [END hopper]
         public static void Main(string[] args)
         {
-            SampleGetBookAsync().Wait();
+            new Parser(with => with.CaseInsensitiveEnumValues = true).ParseArguments<Options>(args)
+                .WithParsed<Options>(opts =>
+                    SampleGetBigBook(opts.Shelf, opts.BigBookName));
```

Comment:
Call the async method, not the sync.
(+quite a few more of this below)

Reply:
Done